torchgeo/trainers/byol.py

```diff
@@ -326,13 +330,18 @@ def __init__(self, **kwargs: Any) -> None:
                 or the path to a saved model state dict.
             learning_rate: Learning rate for optimizer
             learning_rate_schedule_patience: Patience for learning rate scheduler
+            image_size: tuple of height, width that define image size
+                used in BYOL Task
 
         Raises:
             ValueError: if kwargs arguments are invalid
 
         .. versionchanged:: 0.4
            The *backbone_name* parameter was renamed to *backbone*. Change backbone
            support from torchvision.models to timm.
+
+        .. versionchanged:: 0.4.1
```

Comment:
New features shouldn't be added in patch releases. Is this absolutely required? If so, we may have to wait until 0.5 to fix this bug...

Reply:
I could also pretend that I didn't see this and we could document it as 0.5 instead of 0.4.1...

Reply:
The ViT architectures require an input size of 224x224 and the current version it is 256x256 so currently you cannot run BYOL with a pretrained ViT, because that line will always throw an error.